translator/translate/metrics/global_dimensions/globalDimensions_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package global_dimensions
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGlobalDimensions(t *testing.T) {
+	e := new(globalDimensions)
+	var input interface{}
+	err := json.Unmarshal([]byte(`{
+      "global_dimensions": {
+				"Environment": "test",
+				"Dimension": "value",
+				"InvalidBecauseNoValue": "",
+				"": "InvalidBecauseNoKey"
+			}
+    }`), &input)
+	if err == nil {
+		actualKey, actualValue := e.ApplyRule(input)
+		expected := map[string]interface{}{
+			"global_dimensions": map[string]interface{}{
+				"Environment": "test",
+				"Dimension":   "value",
+			},
+		}
+		assert.Equal(t, expected, actualValue, "Expect values to be equal")
+		assert.Equal(t, actualKey, "outputs", "Expect keys to be equal")
+	} else {
+		panic(err)
+	}
```

Comment:
I think rather than doing this if-else, we should just use `assert.NoError(t, err)` and unwrap it. This applies to all of the checks in this test file. In this case, I am not concerned about other test files following this pattern because this needs to be tidied up here (if you saw it elsewhere feel free to point it out. we need to clean it up)

Reply:
Good point @SaxyPandaBear. I've fixed these now.